pixano/app/models/base_schema.py

```diff
@@ -29,16 +31,56 @@ class BaseSchemaModel(BaseModel, Generic[T]):
 
     model_config = ConfigDict(validate_assignment=True)
     id: str
+    created_at: datetime
+    updated_at: datetime
     table_info: TableInfo
     data: dict[str, Any]
 
+    def __init__(self, /, created_at: datetime | None = None, updated_at: datetime | None = None, **data: Any):
+        """Create a new model by parsing and validating input data from keyword arguments.
+
+        Raises [`ValidationError`][pydantic_core.ValidationError] if the input data cannot be
+        validated to form a valid model.
+
+        `self` is explicitly positional-only to allow `self` as a field name.
+
+        Args:
+            created_at: The creation date of the object.
+            updated_at: The last modification date of the object.
+            data: The data of the object validated by Pydantic.
+        """
+        if created_at is None or updated_at is None:
+            if updated_at is not None or created_at is not None:
+                raise ValidationError("Both 'created_at' and 'updated_at' should be set.")
```

Comment:
I suggest we allow a None "updated_at" with a non None "created_at", as at creation time, updated_at == created_at
Badly it means changes to some tests and inits...